### PR TITLE
GFF Blin v1.5.1

### PR DESCRIPTION
**Pull request by dyacob**
This is an update to the included fonts only.  The font dimension properties have been synchronized with Abyssinica for better typography when used together.  Blin "ng" letters have also been refined in many cases for greater clarity.

### CODE REVIEW

**Comment by keyman-server** [comment]
This pull request is from an external repo and will not automatically be built. The build must still be passed before it can be merged. Ask one of the team members to make a manual build of this PR.

File: release/gff/gff_blin/HISTORY.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # ብሊን (Blin) Change History
 
+## 2023-05-07
```
Comment (darcywong00) [inline release/gff/gff_blin/HISTORY.md:3]:
```suggestion
## 2023-05-07 1.51
```

Just wondering if version 1.5.1 would be better than 1.51. (In the other places on the PR too)

**Comment by darcywong00** [comment]
> This is an update to the included fonts only. 

Are you intending the keyboard to use the Waldba font files? If so, we should also update:
1. kps file so the font files are included in the keyboard package, and the keyboards to use Waldba.
2. .keyboard_info file so the the keyboard metadata gets updated.

File: release/gff/gff_blin/HISTORY.md (hunk 1 above)
Comment (mcdurdin) [reply release/gff/gff_blin/HISTORY.md:3]: Yes because 1.6 < 1.51!

**Comment by dyacob** [comment]
Thanks @darcywong00 for catching the version problem, @LornaSIL corrected me on that recently also, so I'm more than a little embarrassed.  I've pushed in fixes.  Regarding the fonts, on my end I see the Waldba fonts in the KPS file list (see screenshot).  They are meant for the desktop version only, so I think a change to the `keyboard_info` file is not needed, since that is for the Keyman Web as I understand it.

WaldbaFontsInGFF_Blin

**Review by LornaSIL — APPROVED** [review]
LGTM.

**Pull request merged**